### ISSUE TITLE
on blur, terminal textarea is cleared so screen reader user cannot review input

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#190998]
1. with a screen reader on, focus the terminal and type some characters
2. use left and right arrows to inspect the input
3. focus something else and then refocus the terminal and try to do the same
4. 🐛 the textarea has been cleared so it announces blank

**Comment by meganrogge** [comment]
the text is in the textarea actually, but not visible and not detected by the screen reader after the blur 🤔 

![Image](https://github.com/microsoft/vscode/assets/29464607/8c7113cf-e24a-4a7b-b669-aacb6315429d)
